packages/components/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,150 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { FocusZone } from '@fluentui-react-native/focus-zone';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, TabsContextData } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings } from './Tabs.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
+import { useSelectedKey } from '@fluentui-react-native/interactive-hooks';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  views: null,
+});
+
+export const Tabs = compose<TabsType>({
+  displayName: tabsName,
+
+  usePrepareProps: (userProps: TabsProps, useStyling: IUseComposeStyling<TabsType>) => {
+    const {
+      label,
+      accessibilityLabel = userProps.label,
+      selectedKey,
+      headersOnly,
+      defaultSelectedKey,
+      getTabId,
+      componentRef = React.useRef(null),
```

Comment:
There was a recent change that fixed the usage of conditional hooks: https://github.com/microsoft/fluentui-react-native/pull/841 

I'd apply the same change here as well.

Reply:
Added that change in.